README.md

```diff
@@ -0,0 +1,5 @@
+# FCC Test Suite Template
```

Comment:
I think adding a custom ReadMe that is related to this project is better than having this generic type of ReadMe

Reply:
Nice review, but it is missing the final summary. 

Please check this link on [how to summarize your review](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md)

Also, Kindly make sure to drop at least two inline comment on this project